lib/pengine/unpack.c

```diff
@@ -106,11 +106,11 @@ pe_fence_node(pe_working_set_t * data_set, pe_node_t * node,
                 crm_notice("Not fencing guest node %s "
                            "(otherwise would because %s): "
                            "its guest resource %s is unmanaged",
-                           node->details->uname, reason, rsc->id);
```

Comment:
I think all these strings could be witten as `"Not fencing guest %s"`, `"Guest %s will be fenced"`, etc. and still be okay.

Reply:
These I'd be more concerned about since they're user-facing, and the message includes both the guest node name and the guest resource name. Both names could be considered just the "guest", but the message has to distinguish them.